configs/recognition/csn/ircsn_ig65m_pretrained_bnfrozen_r152_32x2x1_58e_kinetics400_rgb.py

```diff
@@ -95,7 +95,8 @@
         pipeline=test_pipeline))
 # optimizer
 optimizer = dict(
-    type='SGD', lr=0.0005, momentum=0.9, weight_decay=0.0001)  # 0.0005 for 32g
+    type='SGD', lr=0.0005, momentum=0.9,
+    weight_decay=0.0001)  # this lr is used for 32 gpus
```

Comment:
We should use the LR for 8 gpus in our config files, even if the model is not trained with 8 gpus.